### ISSUE TITLE
Release kedro-sphinx-theme

### ISSUE THREAD

**Issue by astrojuanlu** [kedro-org/kedro-sphinx-theme#11]
Ideally, to fix #4 across our repos.

Should be done after #10.

Stretch goal: #8.

**Comment by astrojuanlu** [comment]
Done! Well done @Huongg and thanks @ankatiyar and @merelcht 👏🏼
